docs/modules/operation/pages/deep-dive/service-assurance/path-outages.adoc

```diff
@@ -1,32 +1,32 @@
 
 [[ga-service-assurance-path-outage]]
-= Path Outages
+= Use Path Outages to Suppress Notifications
 
 An outage of a central network component can cause a lot of node outages.
-Path Outages can be used to suppress notifications based on how nodes depend on each other in the network which are defined in a Critical Path.
-The Critical Path needs to be configured from the network perspective of the monitoring system.
-By default, the Path Outage feature is disabled and has to be enabled in the `poller-configuration.xml`.
+You can use path outages to suppress notifications based on how nodes depend on each other in the network which are defined in a critical path.
+The critical path needs to be configured from the network perspective of the monitoring system.
+By default, the path outage feature is disabled and has to be enabled in `poller-configuration.xml`.
 
-The following image shows an example network topology.
+The following image shows a sample network topology.
 
 [[path-outage-example]]
 .Path Outage example
```

Comment:
Since the previous line describes the image, I don't think we need the alt text.

```suggestion
```